Source/Core/VideoCommon/TextureDecoder_Common.cpp

```diff
@@ -320,7 +320,7 @@ static const char* texfmt[] = {
     "CZ16L", "0x3D", "0x3E", "0x3F",
 };
 
-static void TexDecoder_DrawOverlay(u8* dst, int width, int height, int texformat)
+static void TexDecoder_DrawOverlay(u8* __restrict dst, int width, int height, int texformat)
```

Comment:
It still helps - without __restrict, dst could alias other buffers used by TexDecoder_DrawOverlay. 

Granted, it looks like the only other buffer used by that function is sfont_map, and since sfont_map is a static const array, the compiler might take that to mean writes to dst can't alias sfont_map. I'm not 100% sure, though.